todo/src/App.js

```diff
@@ -1,19 +1,59 @@
-// import logo from './logo.svg';
-import { useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import "./App.css";
+import TodoForm from "./Components/TodoForm";
+import TodoList from "./Components/TodoList";
+import Typography from "@material-ui/core/Typography";
+
+const LOCAL_STORAGE_KEY = "react-todo-list-todos";
 
 function App() {
+  const [todos, setTodos] = useState([]);
 
-  // const [todos, setTodos] = useState(["eat", "sleep", "code", "pray", "repeat"]);  //sample todo items.  these will need to be changed in your app.  Just filler todos, although they are quite important!
+  useEffect(() => {
+    const storageTodos = JSON.parse(localStorage.getItem(LOCAL_STORAGE_KEY));
+    if (storageTodos) {
+      setTodos(storageTodos);
+    }
+  }, []);
 
-  const [todos, setTodos] = useState([]);
+  useEffect(() => {
```

Comment:
There is a lot of functionality in this one JS file.  It would be best if the App.js file were responsible **only** for delivering the final View, **not** for the specific functionality of that View.